src/generated/OpenapiIntegrations.ts

```diff
@@ -31,32 +31,6 @@ export namespace Schemas {
     username?: string | undefined | null;
   };
 
-  export const BehaviorGroup = zodSchemaBehaviorGroup();
```

Comment:
These are hidden from the openapi in current's `notification-backend`. I left them here because I'm using it for the behavior group work.

I'll probably move them to other file to allow the code to compile and play nice with the schema generator.